index.html

```diff
@@ -42,21 +42,25 @@
                         <span class="icon-bar"></span>
                     </button>
 
-                        <!--<a id="nav_home" href="#landing" class="nav-link">
-                            <img class="hint-logo" src="static/img/logo-color-small.png">
-                        </a>-->
+                       
                     </div>
                     <div class="navbar-collapse collapse" id="custom_collapse">
                         <div class=" navbar-right">
                             <ul class="nav navbar-nav">
                                 <li>
                                     <a id="nav_about" href="#about" class="nav-link">ABOUT</a>
                                 </li>
+                                <li>
+                                    <a id="nav_about" href="#schedule" class="nav-link">SCHEDULE</a>
```

Comment:
Element's id should be unique. Give unique ids to all elements. Otherwise, it is invalid markup.